static/auth-client.js

```diff
@@ -2,6 +2,9 @@
 import { createAuthClient } from "https://esm.sh/better-auth@latest/client";
 import { passkeyClient } from "https://esm.sh/better-auth@latest/client/plugins";
 
+/**
+ * @type {import('better-auth/client').ClientOptions} authClient
```

Comment:
This doesn't really work, but I couldn't find a better type.

Reply:
I am removing it, but yes, I would like to see what I am working with. E.g. the `signIn` etc. utlitity.